lamp/templates/lamp-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  # This uses a "fullname" template (see _helpers)
+  # Basing names on .Release.Name means that the same chart can be installed
+  # multiple times into the same namespace.
+  name: {{template "lamp.fullname" .}}
+  labels:
+    app: {{template "lamp.fullname" .}}
+    # The "heritage" label is used to track which tool deployed a given chart.
+    # It is useful for admins who want to see what releases a particular tool
+    # is responsible for.
+    heritage: {{ .Release.Service | quote }}
+    # This makes it easy to search for all components of a release using kubectl.
+    release: {{ .Release.Name | quote }}
+    # This makes it easy to audit chart usage.
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  replicas: {{default 1 .Values.replicaCount | quote}}
+  template:
+    metadata:
+      labels:
+        app: {{template "lamp.fullname" .}}
+        heritage: {{ .Release.Service | quote }}
+        release: {{ .Release.Name | quote }}
+        chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    spec:
+      containers:
+      - name: {{template "lamp.fullname" .}}
+        # We use the Bitnami PHP-FPM Container to run the PHP application
+        # https://hub.docker.com/r/bitnami/php-fpm
+        image: prydonius/php-fpm:7.0.6-r1
```

Comment:
bitnami/bitnami-docker-php-fpm#42 is merged and part of `bitnami/php-fpm:7.0.6-r1`

/cc @prydonius